### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by estebanmual**
In this Milestone I:
make the desktop version following the Figma file structure

### CODE REVIEW

File: 00_Website/assets/css/styles-home-page.css [hunk 1]
```diff
@@ -465,8 +445,761 @@ input::placeholder,
...
+  display: inline-block;
 }
-@media all and (min-width: 1001px) {
 
-} */
+.information-project5 .short {
+  display: none;
+}
+
+@media all and (min-width: 1440px) {
```
Comment (mmsesay) [inline 00_Website/assets/css/styles-home-page.css]: - [x] I am glad to see this media query which will handle breakpoints for large screens. However, I think `1440px` is way too high if we want this website to look great on other devices too. In that light, I will suggest you use a value between `992px` and `1100px`.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Estebanmual , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mmsesay** [comment]
- [x] Your design is really interactive. If you do a comparison between your work and your chosen Figma design, you will notice you do have some margin or padding to the right of your headline section design. The background image should be snapped to the right of the page.

Your Design |
:---------------:|
e2

Figma Design |
:-----------------:|
e1

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @Estebanmual 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**